src/main/java/org/javarosa/xpath/expr/XPathJoinChunkFunc.java

```diff
@@ -0,0 +1,66 @@
+package org.javarosa.xpath.expr;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.DataInstance;
+import org.javarosa.xpath.XPathArityException;
+import org.javarosa.xpath.XPathNodeset;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+public class XPathJoinChunkFunc extends XPathFuncExpr {
```

Comment:
can you add class-level comments to this and `XPathIdCompressFunc` describing their intended usage/purpose